content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,89 @@
 
-Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations
-*WH/Stock/Vegetables* and *WH/Stock/Fruits*.
-
-To manage those locations, we will create putaway rules. To do so, open the *Inventory* app and go
-to :menuselection:`Configuration --> Putaway Rules`. Then, click on create and configure your first
-rule indicating the main location the product will enter before being redirected to the right
-location.
+.. image:: putaway/create-putaway-rules.png
+   :align: center
+   :alt: Create putaway rules for apples and carrots
 
 .. note::
-   The putaway rules can be defined either per product or per product category.
+  It is also possible to create and manage putaway rules for a single product by going to the
+  product page and clicking the :guilabel:`Putaway Rules` smart button at the top of the page. If
+  the button isn't there at first glance, select the :guilabel:`More` button at the top right to
+  view additional configuration options.
 
-.. image:: putaway/putaw2.png
-   :align: center
+Once a putaway rule has been configured, the product it specifies will be automatically routed to
+the :guilabel:`Store to` location upon arriving in the :guilabel:`When product arrives in` location.
+The summary of internal product movements can be viewed by selecting :menuselection:`Reporting -->
+Product Moves` and enabling the :guilabel:`Internal` search filter on the :guilabel:`Filters`
+dropdown under the :guilabel:`search bar` at the top of the page.
 
-Now, if I purchase apples and carrots to my supplier, they will be grouped in the same receipt but
-redirected to the right location automatically, thanks to putaway rules. This information is
-available from *Inventory Report*, under the reporting menu.
 
-.. image:: putaway/putaw3.png
+After configuring the putaway rules for apples and carrots, see the internal stock moves by
+first buying products from a vendor using a :ref:`purchase order<inventory/purchase/vendor bills>`.
+
+In the :guilabel:`Purchase` app, create a :guilabel:`Request for Quotation` in the top of the menu,
+add the products, and select :guilabel:`Confirm` to send the order to the vendor.
+
+.. image:: putaway/purchase-apples-and-carrots.png
    :align: center
+   :alt: Purchase order for apples and carrots
+
+Receive the incoming shipment by selecting :menuselection:`Receive products --> Validate`.
 
-.. image:: putaway/putaw4.png
+.. image:: putaway/receive-incoming-stock.png
    :align: center
+   :alt: Receive package of apples and carrots
+
+Verify stock moves
+==================
+
+Confirm whether the products have been moved to the correct location, go to the
+:guilabel:`Inventory` app, and in the upper menu, navigate to :menuselection:`Reporting -->
+Inventory Report` to view the all products in stock. By default, the products are grouped by
+location. Select the record for a specific product to view the location where the products are
+stored.
+
```

Comment:
I think we could use a bit more detail in this section about reporting. Like, what are some other cool ways we can group our products to find stuff, and what can we group? How granular can we get?

**Also** I think `locations` might deserve a `:guilabel:` at a few points in this section but I'll defer to Zac's final say on that one.